lib/TokenManager.ts

```diff
@@ -116,24 +119,9 @@ function get(storage, key) {
 }
 
 function getAsync(sdk, tokenMgmtRef, storage, key) {
-  return new Promise(function(resolve, reject) {
-    if (tokenMgmtRef.options.autoRenew && isLoginRedirect(sdk)) {
-      return reject(new AuthSdkError(
-        'The app should not attempt to call authorize API on callback. ' + 
-        'Authorize flow is already in process. Use parseFromUrl() to receive tokens.'
-      ));
-    }
-
+  return new Promise(function(resolve) {
```

Comment:
Do we still need to keep this method async? Looks like all async logic has been removed.

Reply:
We have a long-standing feature request to provide async option for custom storage provider: https://oktainc.atlassian.net/browse/OKTA-269814 The use case here is if you store tokens in a database. If we intend to suppor this, it is a good idea to keep get/set as async.